src/Sign.SignatureProviders.TrustedSigning/TrustedSigningService.cs

```diff
@@ -69,12 +67,13 @@ public async Task<X509Certificate2> GetCertificateAsync(CancellationToken cancel
 
                     _logger.LogTrace(Resources.FetchingCertificate);
 
-                    CertificateProfileSignOperation operation = await _client.StartSignAsync(_accountName, _certificateProfileName, _emptyRequest, cancellationToken: cancellationToken);
-                    Response<SignStatus> response = await operation.WaitForCompletionAsync(cancellationToken);
+                    var response = await _client.GetSignCertificateChainAsync(_accountName, _certificateProfileName, cancellationToken: cancellationToken);
+
+                    var rawData = new MemoryStream();
```

Comment:
```suggestion
                    MemoryStream rawData = new();
```

Reply:
I'm going to refactor this to use byte[] directly from the stream.